docs/topics/time.md

```diff
@@ -0,0 +1,373 @@
+[//]: # (title: Time)
+
+The Kotlin standard library gives you the tools you need to be able to calculate and measure time in different units.
+Being able to measure time accurately is important for activities like:
+  * Managing threads or processes
+  * Collecting statistics
+  * Detecting timeouts
+  * Debugging
+
+By default, time is measured using a monotonic time source but other time sources can be configured.
+
+## Durations
+
+To represent an amount of time, the standard library has the [`Duration`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.time/-duration/)
+class. A `Duration` can be expressed in the following units from the [`DurationUnit`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.time/-duration-unit/)
+Enum class:
+  * `NANOSECONDS`
+  * `MICROSECONDS`
+  * `MILLISECONDS`
+  * `SECONDS`
+  * `MINUTES`
+  * `HOURS`
+  * `DAYS`
+
+`Durations` can also be zero, negative or infinite.
+
+### Calculate duration
+
+To create an instance of the `Duration` class, use the [extension properties](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.time/-duration/#companion-object-properties) 
+available for `Int`, `Long`, and `Double` types: `nanoseconds`, `microseconds`, `milliseconds`, `seconds`, `minutes`, 
+`hours`, and `days`.
+
+For example:
+
+```kotlin
+import kotlin.time.*
+import kotlin.time.Duration.Companion.milliseconds
+import kotlin.time.Duration.Companion.seconds
+import kotlin.time.Duration.Companion.minutes
+
+fun main() {
+    //sampleStart
+    val fiveHundredMilliseconds: Duration = 500.milliseconds
+    val fiveSeconds: Duration = 5.seconds
+    val tenMinutes: Duration = 10.minutes
+
+    println(fiveHundredMilliseconds) // 500ms
+    println(fiveSeconds)             // 5s
+    println(tenMinutes)              // 10m
+    //sampleEnd
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3"}
+
+To create a `Duration` of zero seconds, use the `ZERO` property:
```

Comment:
`ZERO` may be a convenient constant, but I wouldn't give a strong recommendation to use it certainly if you need a duration of zero seconds.

Reply:
OK, I'll remove this section.